<launch>
	<!-- Load the parameters file -->
	<rosparam command="load" file="$(find r_platform)/params/default.yaml"/>
	<!-- joy node -->
	<node respawn="true" pkg="joy" type="joy_node" name="joy_node">
		<param name="dev" type="string" value="/dev/input/js0" />
		<param name="deadzone" value="0.12" />
		<param name="autorepeat_rate" value="20" />
	</node>
	<node pkg="r_platform" type="r_hybrid" name="r_hybrid" />

	<node pkg="r_platform" type="decompose_vel" name="decompose_vel" />
	<node pkg="r_platform" type="r_client_node" name="r_client_node" />
	<!-- <node pkg="hokuyo_node" type="hokuyo_node" name="hokuyo_node"/> -->


	<node name="hokuyo0" pkg="hokuyo_node" type="hokuyo_node">
		<param name="port" type="string" value="/dev/ttyACM0" />
		<param name="frame_id" value="/laser1"/>
		<param name="min_ang" type="double" value="-1.9" />
		<param name="max_ang" type="double" value="1.9" />
		<param name="skip" type="int" value="0" />
		<param name="intensity" value="false" />
		<remap from="scan" to="scan1" />
	</node>
	<node name="hokuyo1" pkg="hokuyo_node" type="hokuyo_node">
		<param name="port" type="string" value="/dev/ttyACM1" />
		<param name="frame_id" value="/laser"/>
		<param name="min_ang" type="double" value="-2.35619449019234492883" />
		<param name="max_ang" type="double" value="2.35619449019234492883" />
		<param name="skip" type="int" value="0" />
		<param name="intensity" value="false" />
		<remap from="scan" to="scan0" />
	</node>
</launch>
